docs/platforms/powershell/configuration/draining.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Shutdown and Draining
+sidebar_order: 70
+description: "Learn more about the default behavior of our SDK if the application shuts down unexpectedly."
+notSupported:
+  - dart
+  - ruby
+  - unity
+  - php
+  - elixir
+  - unreal
```

Comment:
Do we need this? now there's no inheritance of docs anymore, this is specific to powershell right?

```suggestion
```